src/overlays/actors/ovl_Magic_Wind/z_magic_wind.c

```diff
@@ -15,39 +15,244 @@ void MagicWind_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void MagicWind_Update(Actor* thisx, GlobalContext* globalCtx);
 void MagicWind_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
 const ActorInit Magic_Wind_InitVars = {
-    ACTOR_MAGIC_WIND,
-    ACTORTYPE_ITEMACTION,
-    FLAGS,
-    OBJECT_GAMEPLAY_KEEP,
-    sizeof(MagicWind),
-    (ActorFunc)MagicWind_Init,
-    (ActorFunc)MagicWind_Destroy,
-    (ActorFunc)MagicWind_Update,
-    (ActorFunc)MagicWind_Draw,
+    ACTOR_MAGIC_WIND,  ACTORTYPE_ITEMACTION, FLAGS,          OBJECT_GAMEPLAY_KEEP, sizeof(MagicWind), MagicWind_Init,
+    MagicWind_Destroy, MagicWind_Update,     MagicWind_Draw,
```

Comment:
Keep the (ActorFunc) cast for now. I don't know if we will be omitting in the future, but for now we're leaving it.